ADD: transform affects DIV child contents

leebut opened the pull request:
After spending some time figuring out how to prevent text in a div being skewed as well as the div, I think it is worth mentioning that applying a transform to a div element will also affect elements contained inside of it. Perhaps the transform challenge pages could include how to negate the effects of the transform on child elements of divs.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

leebut committed:
ADD: transform affects DIV child contents

After spending some time figuring out how to prevent text in a div being skewed as well as the div, I think it is worth mentioning that applying a transform to a div element will also affect elements contained inside of it. Perhaps the transform challenge pages could include how to negate the effects of the transform on child elements of divs.

RandellDawson commented on curriculum/challenges/english/01-responsive-web-design/applied-visual-design/use-the-css-transform-scale-property-to-scale-an-element-on-hover.english.md:
```diff
@@ -15,6 +15,8 @@ Here's an example to scale the paragraph elements to 2.1 times their original si
 <section id='instructions'>
 Add a CSS rule for the <code>hover</code> state of the <code>div</code> and use the <code>transform</code> property to scale the <code>div</code> element to 1.1 times its original size when a user hovers over it.
+  
+  **NOTE:** Applying a transform to a <code>div</div> element will also affect any child elements contained in the div.
```
Since this is more information, maybe it should be in the Description section and not the Instructions section?

leebut replied:
Good point. Thank you.

leebut committed:
Fix: moved my commit to Description 

Moved my commit text into the Description section as suggested.

thecodingaviator commented on curriculum/challenges/english/01-responsive-web-design/applied-visual-design/use-the-css-transform-scale-property-to-scale-an-element-on-hover.english.md:
```diff
@@ -10,6 +10,8 @@ videoUrl: 'https://scrimba.com/c/cyLPJuM'
 Here's an example to scale the paragraph elements to 2.1 times their original size when a user hovers over them:
 <blockquote>p:hover {<br>&nbsp;&nbsp;transform: scale(2.1);<br>}</blockquote>
+   
+  **NOTE:** Applying a transform to a <code>div</div> element will also affect any child elements contained in the div.
```
```suggestion
  **NOTE:** Applying a transform to a <code>div</code> element will also affect any child elements contained in the div.
```

RandellDawson replied:
@thecodingaviator Good catch!

leebut replied:
Ah, yes, just noticed it. I've been typing many divs. Thank you for changing it.

RandellDawson replied:
@leebut You still need to change it.  @thecodingaviator just made a suggestion.

leebut replied:
Okay. Done. Thank you.

leebut committed:
Fix: Changed </div> to </code>

Changed the closing </div> tag to </code> as suggested.

leebut committed:
Fix: Changed markdown ** to <strong>

As advised on a previous pull request, I have removed the markdown ** ** and replaced it with HTML <strong></strong>.

RandellDawson commented:
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉
